### PR TITLE
Feature/my orders

### PR DESCRIPTION
**Pull request by travisdmathis**
##### Checklist

- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,11 +51,12 @@
     "dotenv-expand": "4.2.0",
     "ethereum-input-data-decoder": "0.0.12",
     "extract-text-webpack-plugin": "3.0.2",
-    "file-loader": "2.0.0",
-    "fs-extra": "7.0.0",
-    "html-webpack-plugin": "3.2.0",
-    "human-standard-token-abi": "^2.0.0",
-    "jest": "23.6.0",
+    "file-loader": "1.1.5",
```
Comment (eswarasai) [inline package.json]: @travisdmathis -- Reason to revert these packages version from that of `develop`? Did you find any breaking changes that I've missed?

Comment (travisdmathis) [reply package.json]: could be that you didn't run npm install after you upgraded them and uploaded a mismatched package-lock file, so when i ran npm install it installed older versions?

Comment (eswarasai) [reply package.json]: Yeah. Looks like rebase issues. Would be great if you can just revert them to the latest versions as below. Thanks!

```
     "file-loader": "2.0.0",
    "fs-extra": "7.0.0",
    "html-webpack-plugin": "3.2.0",
    "human-standard-token-abi": "^2.0.0",
    "jest": "23.6.0",
```

Comment (travisdmathis) [reply package.json]: was mistake, had a conflict and when i corrected it, reverted them

**Pull request merged**